src/ripple/app/misc/Validations.cpp

```diff
@@ -387,6 +387,31 @@ class ValidationsImp : public Validations
         return ret;
     }
 
+    std::uint32_t getValidationTime (uint256 const& hash, int minValidations)
+    {
+        if (minValidations < 1)
+            minValidations = 1;
+
+        std::vector <std::uint32_t> times;
+
```

Comment:
@vinniefalco I agree that it makes sense to pick the best reserve value we have readily available.  If the total number of validators that might contain the requested hash is available, that's certainly a better choice than `minValidations`. If we use `minValidations` there might be more reallocations coming.  But I figured it was better than no initial reserve.